training/README.md

```diff
@@ -1,15 +1,21 @@
-Bare metal bootc containers
+Linux Operating System Bootable containers enabled for AI Training
 ===
 
 In order to run accelerated AI workloads, we've prepared [bootc](https://github.com/containers/bootc) container images for the major AI platforms.
 
 # Makefile targets
 
-| Target      | Description                                             |
-|-------------|---------------------------------------------------------|
-| amd         | Create bootable container for AMD platform              |
-| nvidia      | Create bootable container for NVidia platform           |
-| intel       | Create bootable container for Intel Habanalabs platform |
+| Target          | Description                                                         |
+|-----------------|---------------------------------------------------------------------|
+| instruct-amd    | Create instruct lab image for bootable container for AMD platform   |
+| instruct-intel  | Create instruct lab image for bootable container for Intel platform |
+| instruct-nvidia | Create instruct lab image for bootable container for Nvidia platform|
+| amd             | Create bootable container for AMD platform                          |
+| intel           | Create bootable container for Intel Habanalabs platform             |
+| nvidia          | Create bootable container for NVidia platform                       |
+| cloud-amd       | Add cloud-init to bootable container for AMD platform               |
```

Comment:
It feels like it'd be nicer to have this as flag, something like

```
make amd INCLUDES=cloud-init
```

Would generalize more obviously to including other snippets in the future.

Reply:
Sounds good lets merge and then I can work on this. Would involve more complicated Makefile gymnastics.